pkgs/tools/text/gtranslator/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, fetchurl
+, meson
+, ninja
+, pkgconfig
+, itstool
+, python3
+, wrapGAppsHook
+, libxml2
+, libgda
+, gspell
+, glib
+, gtk3
+, gtksourceview4
+, gnome3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gtranslator";
+  version = "3.32.0";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "1dqigah7x6h3afw5jcs3lw3h9y0acj72arcgxr89l6galvknr2xa";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    itstool
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    libxml2
+    glib
+    gtk3
+    gtksourceview4
+    libgda
+    gspell
+    gnome3.gsettings-desktop-schemas
```

Comment:
```suggestion
    gsettings-desktop-schemas
```

Reply:
This is just an alias.

Reply:
Ahh
```
inherit (gnome3) gsettings-desktop-schemas;
```

Could we move that out?

Reply:
We probably should. And same with other `inherited`s.

Reply:
https://github.com/NixOS/nixpkgs/pull/59750 :smile: